types/zrender/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Don't disable any rules unless there's a valid scenario.

Reply:
I'm agree, sorry for that. It was just copied from `echarts`. I'll fix that

Reply:
Done.